### DISCUSSION TITLE
Invalid Add-on repository!

### DISCUSSION THREAD

**Discussion by terasankka** [alandtse/tesla#17] [Q&A] [answered]
Any tips how to get this added. I get "Invalid Add-on repository!" when I try to add this " https://github.com/alandtse/tesla " -custom repository?

**Comment by alandtse** [comment]
You don't have to add it. It's a default repo.

**Reply by alandtse** [reply]
Oops sorry. It thought I had added it. You have to add it in [HACS](https://hacs.xyz/). Not the Home Assistant addons.

**Reply by CobraDunn** [reply]
Same thing here when adding HACS.

**Comment by N3W4L** [comment]
I got the same issue today, how did you fix this? thanks

**Reply by Leytronix** [reply]
Me too. Is there already a solution? Thanks Tjerk

**Comment by alandtse** [comment, marked as answer]
You have to add it in [HACS](https://hacs.xyz/). Not the Home Assistant addons.
